README.md

```diff
@@ -57,6 +57,12 @@ This is a project to build a blog app with Ruby on Rails. The main goal is to le
 
 - **Creating a data model**
 - **Processing data in models**
+- **Validations and Model specs**
+- **Setup and controllers**
+- **Controllers specs**
+- **Views**
+- **Forms**
+- **Integration specs for Views and fixing n+1 problems**
```

Comment:
- [ ] These changes are being requested here because the affected files cannot be highlighted, but these prevent the project from being run. When running the migration for your project I get these errors due to the names of the classes in your 4 migration files in the `db/migrate` directory. Your respective classes have the names

  ```
  BlogUsers, BlogPosts, BlogComments and BlogLikes
  ```
  But they should have these instead. Kindly rename the class names.

  ```
  CreateUsers, CreatePosts, CreateComments, CreateLikes
  ```

  ![image](https://github.com/sergio-usma/rails_blog_pj/assets/81468675/0dda8408-61e6-4fa2-ae47-9b04f9d464b2)